include/model.h

```diff
@@ -63,6 +66,7 @@ class Model{
     void debug();
   private:
     std::string calc_time; // stores the time when the calculation was run for output folder and report
+    std::map<char,double> volumes_stored; // convenient access to calculated volumes for report

```

Comment:
CalcResultBundle already contains this exact map. Store an instance of CalcResultBundle instead